{
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
        "ID": "CVE-2019-3871",
        "ASSIGNER": "psampaio@redhat.com"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "The PowerDNS Project",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "pdns",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "4.1.7"
                                        },
                                        {
                                            "version_value": "4.0.7"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-20"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3871",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-3871",
                "refsource": "CONFIRM"
            },
            {
                "url": "https://doc.powerdns.com/authoritative/security-advisories/powerdns-advisory-2019-03.html"
            }
        ]
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "A vulnerability was found in PowerDNS Authoritative Server before 4.0.7 and before 4.1.7. An insufficient validation of data coming from the user when building a HTTP request from a DNS query in the HTTP Connector of the Remote backend, allowing a remote user to cause a denial of service by making the server connect to an invalid endpoint, or possibly information disclosure by making the server connect to an internal endpoint and somehow extracting meaningful information about the response"
            }
        ]
    },
    "impact": {
        "cvss": [
            [
                {
                    "vectorString": "6.5/CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:L",
                    "version": "3.0"
                }
            ]
        ]
    }
}
